ansible/configs/ansible-tower-implementation/default_vars_ec2.yml

```diff
@@ -0,0 +1,161 @@
+################################################################################
+### Environment Settings for aws
+################################################################################
+## Environment Sizing
+
+cloud_provider: ec2     # Which AgnosticD Cloud Provider to use              
+### Route 53 Zone ID (AWS)
+# This is the Route53 HostedZoneId where you will create your Public DNS entries
+# This only needs to be defined if your CF template uses route53
+HostedZoneId: Z3IHLWJZOU9SRT
+aws_region: ap-southeast-2
+
+ansible_user: ec2-user
+remote_user: ec2-user
+
+## Environment Sizing
+bastion_instance_type: "t2.medium"
```

Comment:
Not required, but consider moving to t3 to be more future proof.

Reply:
changed to t3